tests/roxctl/bats-tests/local/roxctl-netpol-connectivity-map-development.bats

```diff
@@ -44,16 +54,101 @@ teardown() {
+  cp "${test_data}/np-guard/netpols-analysis-example-minimal/backend.yaml" "$out_dir/backend.yaml"
+  cp "${test_data}/np-guard/netpols-analysis-example-minimal/frontend.yaml" "$out_dir/frontend.yaml"
+  cp "${test_data}/np-guard/netpols-analysis-example-minimal/netpols.yaml" "$out_dir/netpols.yaml"
+  write_yaml_to_file "$templated_fragment" "$(mktemp "$out_dir/templated-01-XXXXXX-file1.yaml")"
+  write_yaml_to_file "$templated_fragment" "$(mktemp "$out_dir/templated-02-XXXXXX-file2.yaml")"
+
+  run roxctl-development netpol connectivity map "$out_dir/" --fail --strict
+  assert_failure
+  assert_output --partial 'there were warnings during execution'
+  assert_output --partial 'file1.yaml'
+  refute_output --partial 'file2.yaml'
+
+  run roxctl-development netpol connectivity map "$out_dir/"
+  assert_success
+  assert_output --partial 'there were warnings during execution'
+  assert_output --partial 'file1.yaml'
+  assert_output --partial 'file2.yaml'
+
+  run roxctl-development netpol connectivity map "$out_dir/" --fail
+  assert_success
+  assert_output --partial 'there were warnings during execution'
+  assert_output --partial 'file1.yaml'
+  assert_output --partial 'file2.yaml'  
+
+  run roxctl-development netpol connectivity map "$out_dir/" --strict
+  assert_failure
+  assert_output --partial 'there were warnings during execution'
+  assert_output --partial 'file1.yaml'
+  assert_output --partial 'file2.yaml'    
+}
+
+# TODO: does this test represent a desired behavior? 
+# @test "roxctl-development netpol connectivity succedes and returns parital warnings when run with --fail" {
+#   mkdir -p "$out_dir"
+#   assert_file_exist "${test_data}/np-guard/netpols-analysis-example-minimal/backend.yaml"
+#   assert_file_exist "${test_data}/np-guard/netpols-analysis-example-minimal/frontend.yaml"
+#   assert_file_exist "${test_data}/np-guard/netpols-analysis-example-minimal/netpols.yaml"
+#   cp "${test_data}/np-guard/netpols-analysis-example-minimal/backend.yaml" "$out_dir/backend.yaml"
+#   cp "${test_data}/np-guard/netpols-analysis-example-minimal/frontend.yaml" "$out_dir/frontend.yaml"
+#   cp "${test_data}/np-guard/netpols-analysis-example-minimal/netpols.yaml" "$out_dir/netpols.yaml"
+#   write_yaml_to_file "$templated_fragment" "$(mktemp "$out_dir/templated-01-XXXXXX-file1.yaml")"
+#   write_yaml_to_file "$templated_fragment" "$(mktemp "$out_dir/templated-02-XXXXXX-file2.yaml")"
+
+#   run roxctl-development netpol connectivity map "$out_dir/" --fail
+#   assert_success
+#   assert_output --partial 'there were warnings during execution'
+#   assert_output --partial 'file1.yaml'
+#   refute_output --partial 'file2.yaml'
+
+#   run roxctl-development netpol connectivity map "$out_dir/"
+#   assert_success
+#   assert_output --partial 'there were warnings during execution'
+#   assert_output --partial 'file1.yaml'
+#   assert_output --partial 'file2.yaml'
+# }
```

Comment:
I kept this commented-out test just to demonstrate a behavior I think is not desired. 
The change of the condition for running the builder with `continueOnError` that I've made, has addressed this issue.